specs/interop/predeploys.md

```diff
@@ -245,7 +266,11 @@ function relayMessage(uint256 _destination, uint256 _source, uint256 _nonce, add
        _calldata: _message
     });
 
-    require(success);
+    if (!success) {
+      if (failedMessages[messageHash] == 0) {
```

Comment:
This branch needs to exist so that somebody cannot replay over and over again, making the timestamp in storage go up and up, preventing a rollback. Can you note this?